modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -62,7 +62,7 @@ $ gadmin config set System.CrossRegionReplication.PrimaryKafkaIPs <PRIMARY_IP1,P
 $ gadmin config set System.CrossRegionReplication.PrimaryKafkaPort 30002
 ----
 +
-.The prefix of GPE/GUI/GSQL Kafka Topic, by default is empty.
+.The prefix of GPE/GUI/GSQL Kafka Topic, by default is empty. For setting up new DR after failover, refer to xref:tigergraph-server:cluster-and-ha-management:fail-over.adoc#_set_up_a_new_dr_cluster_after_failover[].
```

Comment:
Following changes were made in the previous PR that were made but not reflected on the web page